docs/ManuallyRunning.md

```diff
@@ -0,0 +1,237 @@
+```shell script
+$ export SELECTED_TESTS_URI=http://localhost:8080
+$ export SELECTED_TESTS_PROJECT=mongodb-mongo-master 
+$ curl  -X POST ${SELECTED_TESTS_URI}/projects/${SELECTED_TESTS_PROJECT}/test-mappings \
+        -d '{"source_file_regex":"^src/mongo", "test_file_regex":"^jstests"}' \
+        -H "Content-Type: application/json"
+$ curl -X POST ${SELECTED_TESTS_URI}/projects/${SELECTED_TESTS_PROJECT}/task-mappings \
+       -H 'accept: application/json' \
+       -H 'Content-Type: application/json' \
+       -d '{ "source_file_regex": "^src/mongo", "module": "enterprise", "module_source_file_regex": "^src", "build_variant_regex": "^!"}'
+``` 
+
+Now that the mappings have been created, we must populate them.
+
+## Populating the mappings.
+ 
+You should run the `process-test-mappings` and `process-task-mappings` commands once every
+day (see [Generate test and task mappings](#generate-test-task-mappings)) . This will gather the
+unprocessed test mapping create and task mapping create requests and process them so that test and
+task mappings for those projects are added to the db.
+
+You should run the `test-mappings update` and `task-mappings update` daily to update the test and
+task mappings models. These jobs look at all git commits and mainline patch builds from the previous
+day and create new test mappings and task mappings respectively.
+
+Assuming your [github keys](#github-access) were created and are in  **~/.ssh/selected_tests_rsa**,
+you can run the following commands.
+
+```shell script
+$ eval "$(ssh-agent -s)"
+$ ssh-add -k ~/.ssh/selected_tests_rsa
+$ export EVG_API_USER=$(python -c "import yaml; import sys; cfg = yaml.safe_load(sys.stdin); print(cfg['user'])" < ~/.evergreen.yml)
+$ export EVG_API_KEY=$(python -c "import yaml; import sys; cfg = yaml.safe_load(sys.stdin); print(cfg['api_key'])" < ~/.evergreen.yml)
+$ export SELECTED_TESTS_URI=http://localhost:8080
+
+# The following commands may take some time to run.
+$ poetry run test-mappings --log-format json update
+$ poetry run work-items --log-format json process-test-mappings
+
+# The following commands may take some time to run.
+$ poetry run task-mappings --log-format json update
+$ poetry run work-items --log-format json process-task-mappings
+```
+
+As noted, the test / task mapping update and test / task work-item commands should be run daily to
+ensure that your mapping are kept up to date. 
+
+# View Selected Tests Service mappings 
+
+You can use the swagger access page or the command line to view the Selected Tests Service Mappings.
+
+# Swagger 
+
+You can view the mappings on the [swagger page](http://localhost:8080/swagger) for the web-service.
+
+This assumes that the service is running on **http://localhost:8080**.
+
+See [Selected Test Swagger API](../README.md#accessing-selected-test-api).
+
+Documentation for how the swagger documentation is done can be found
+[here](https://flask-restplus.readthedocs.io/en/stable/swagger.html).
+
+## Curl Mappings 
```

Comment:
Since people can use whatever tool they want to hit the API (curl, Postman, etc.), and since Swagger's purpose is to explain how people can use our API endpoints, I think it is out of scope to include the curl commands here